vendor/modules.txt

```diff
@@ -2,6 +2,9 @@
 # bitbucket.org/bertimus9/systemstat v0.5.0
 ## explicit; go 1.17
 bitbucket.org/bertimus9/systemstat
+# dario.cat/mergo v1.0.0
+## explicit; go 1.13
+dario.cat/mergo
```

Comment:
weird, where did this come from?

Reply:
looks like all the verify jobs are green, so not my IDE

Reply:
yeah, I didn't realize we were still using the mergo library... if we are, it makes sense it is here